docs/prod-host-setup.md

```diff
@@ -1,5 +1,28 @@
 # Production Host Setup Recommendations
 
+## Jailer Configuration
+
+Using Jailer in a production Firecracker deployment is highly recommended,
+it will provide additional security boundaries for the Firecracker.
+The Jailer process applies [cgroup](https://www.kernel.org/doc/Documentation/cgroup-v1/cgroups.txt)
+, namespace, [seccomp](https://www.kernel.org/doc/Documentation/prctl/seccomp_filter.txt)
+isolation and drops privileges of the Firecracker process.
+
```

Comment:
The two subsequent paragraphs looks like steps the user should take. Can you mark these with something line

```
To set up the jailer correctly, you'll need to:
```

... and then use a list for the steps.

Reply:
Done in latest commit. Thanks